packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,82 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+
+ * @prop --calcite-input-date-picker-background-color: Specifies the component's background color.
+ * @prop --calcite-input-date-picker-border-color: Specifies the component's border color.
+ * @prop --calcite-input-date-picker-shadow: Specifies the component's shadow.
+
+ * @prop --calcite-input-date-picker-divider-background-color: Specifies the component's divider background color.
```

Comment:
The divider is a single line, so this should probably be `--calcite-input-date-picker-divider-color`. This also aligns with [`text-area`'s similar token](https://developers.arcgis.com/calcite-design-system/components/text-area/#api-reference-styles---calcite-text-area-divider-color).